tests/foreman/cli/test_subscription.py

```diff
@@ -191,7 +191,7 @@ def test_positive_manifest_history(self):
         self._upload_manifest(self.org['id'])
```

Comment:
I believe part of pytest conversion could be refactoring internal methods like that into fixtures.

Not to mention that there's already `upload_manifest` method defined in TestCase class, so this one shouldn't be needed.